web-app/src/components/HolidayModal/container.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { isEmpty } from 'lodash';
+
+export default Wrapped =>
+  class extends Component {
+    static propTypes = {
+      holiday: PT.object.isRequired,
+    };
+
+    constructor(props) {
+      super(props);
+      this.state = {};
+    }
+
+    approveHoliday = holidayId => {
+      console.log('approved');
+    };
+
+    rejectHoliday = holidayId => {
+      console.log('rejected');
+    };
+
+    render() {
+      const { holiday } = this.props;
+      if (isEmpty(holiday)) return null;
+      return (
+        <Wrapped
+          {...this.props}
```

Comment:
I'm not a fan of using the spread operator for passing this.props and this.state down as props. Easier to read if explicit?

Reply:
Good spot. Agreed.